nvflare/fuel/f3/cellnet/utils.py

```diff
@@ -93,3 +99,38 @@ def decode_payload(message: Message, encoding_key=MessageHeaderKey.PAYLOAD_ENCOD
         # assume to be bytes
         pass
     message.remove_header(encoding_key)
+
+
+def format_size(num, binary=False):
```

Comment:
What if the "num" passed in is NOT a integer?
Should we check for that?

Reply:
Added support for numbers in string.